### ISSUE TITLE
There are stars

### ISSUE THREAD

**Issue by lizadaly** [NaNoGenMo/2022#35]
[The code](https://github.com/lizadaly/there-are-stars) for this is complete, but the story is unlikely to make 50,000 words before the deadline. Maybe I'll be surprised!

_There are stars_ writes more of its story every time someone stars its Github repo. Each star adds about 50 words of text. Every 30 minutes a Github Action job [publishes the story](https://lizadaly.github.io/there-are-stars/) with the current number of stars.

Right now, it looks like this:

<img width="831" alt="image" src="https://user-images.githubusercontent.com/19571/203162277-0a83ef8c-bd89-49b6-9544-d55c2893ea9d.png">

As text:

> Prologue
> You started this journey over a day ago. You left from your village. It was a rainy day.
> 
> To help others follow you, you lay behind you a winding path of catcastle cobblestones until your pockets were empty of stones.
> 
> Chapter 1
> 21 November
> In a village along the way, a tuner felt a stirring and decided to follow the catcastle path. They set out on a rainy day.
> 
> In an inlet along the way, a head cook observed one new star in the sky and decided to follow the catcastle path. They set out on a wind-swept day.
> 
> After a time, the head cook noticed that the path had become faded and hard to follow. They started a new trail with delicate plantings of tansy flowers until their pockets were empty of seeds.
> 
> Epilogue
> After a long journey, your friends arrived at your sanctuary by the sea. This is how there came to be two new stars in the sky.

Each user who stars is represented by a new person joining the trip. The random occupations and places associated with each person are seeded by the Github username of the starring account, so they're stable between each invocation.

**Comment by lizadaly** [comment]
The final version at the close of business (or just slightly after) was 11,932 words, from 355 stars.

No `completed` label for me but thank you to everyone who participated!

**Comment by dariusk** [comment]
I was the very last star 🥺
